client/components/mma/cancel/CancellationJourneyFunnel.tsx

```diff
@@ -36,9 +33,27 @@ export const CancellationJourneyFunnel = () => {
 		return <Navigate to="/" />;
 	}
 
+	const productType = getSpecificProductTypeFromTier(productDetail.tier);
+	const productTypeKey = productType.productType;
+
+	const possiblePaidPlan = productDetail.subscription.currentPlans[0];
+	const qmEventId = 184;
+	const qmIsConversionEvent = 0;
+	const qmEvent = [
+		'cancellation start',
+		productType.friendlyName,
+		isPaidSubscriptionPlan(possiblePaidPlan) &&
+			`billing period: ${possiblePaidPlan.billingPeriod}`,
+	].filter(Boolean);
```

Comment:
so filter(Boolean) will filter out anything that is not a boolean?

Reply:
anything that is not truthy 👍